tests/test_mdib.py

```diff
@@ -64,6 +64,87 @@ def test_get_descriptor_by_code(self):
+                                                </pm:ConceptDescription>
+                                            </pm:Type>
+                                            <pm:Argument>
+                                                <pm:ArgName Code="codeForArgumentName"></pm:ArgName>
+                                                <pm:Arg {2}>{3}duration</pm:Arg>
+                                            </pm:Argument>
+                                        </pm:Operation>
+                                    </pm:Sco>
+                                    <pm:SystemContext Handle="SC.mds0" DescriptorVersion="2">
+                                        <ext:Extension>
+                                        </ext:Extension>
+                                        <pm:PatientContext Handle="PC.mds0" DescriptorVersion="2">
+                                        </pm:PatientContext>
+                                        <pm:LocationContext Handle="LC.mds0" DescriptorVersion="2">
+                                        </pm:LocationContext>
+                                    </pm:SystemContext>
+                                </pm:Mds>
+                            </pm:MdDescription>
+                        </msg:Mdib>
+                    </msg:GetMdibResponse>"""
+
+        my_prefix = "my"
+        xsd_prefix = "xsd"
+        delaration = 'xmlns:{0}="http://www.w3.org/2001/XMLSchema"'
+        delaration_any_uri = 'xmlns:{0}="urn:oid:1.23.3.123.2"'
+        expected_qname_xsd = QName("http://www.w3.org/2001/XMLSchema", "duration")
+        expected_qname_any_uri = QName("urn:oid:1.23.3.123.2", "duration")
+
+        mdibs = [TestData(mdib_text=mdib_dummy.format('', '', delaration.format(my_prefix), f"{my_prefix}:"),
+                          expected_qname=expected_qname_xsd),
+                 TestData(mdib_text=mdib_dummy.format('', delaration.format(my_prefix), '', f"{my_prefix}:"),
+                          expected_qname=expected_qname_xsd),
+                 TestData(mdib_text=mdib_dummy.format(delaration.format(my_prefix), '', '', f"{my_prefix}:"),
+                          expected_qname=expected_qname_xsd),
+                 TestData(mdib_text=mdib_dummy.format('', '', delaration.format(xsd_prefix), f"{xsd_prefix}:"),
+                          expected_qname=expected_qname_xsd),
+                 TestData(mdib_text=mdib_dummy.format('', '', 'xmlns="http://www.w3.org/2001/XMLSchema"', ''),
+                          expected_qname=expected_qname_xsd),
+                 TestData(mdib_text=mdib_dummy.format('', '', delaration_any_uri.format(xsd_prefix), f"{xsd_prefix}:"),
+                          expected_qname=expected_qname_any_uri),
+                 TestData(mdib_text=mdib_dummy.format('', delaration_any_uri.format(xsd_prefix), '', f"{xsd_prefix}:"),
+                          expected_qname=expected_qname_any_uri)]

```

Comment:
would suggest to add:
TestData(mdib_text=mdib_dummy.format('xmlns="urn:oid:1.23.3.123.2"', '',  '', ''),
                          expected_qname=expected_qname_any_uri)
                          
 line 144
             prefix = arg_node.text.split(':')[0] if ":" in arg_node.text else None

Reply:
done